engine/jsb-spine-skeleton.js

```diff
@@ -138,7 +138,8 @@
 
     Object.defineProperty(skeleton, "timeScale", {
         get () {
-            return this._timeScale || 1.0;
+            // this._timeScale || 1.0 will return 1.0 if this._timeScale is 0
+            return null != this._timeScale ? this._timeScale : 1.0;
```

Comment:
Thanks for finding this issue.
But I think the protection should be added in the setter to avoid any illegal value being set to `this._timeScale`